Library/Homebrew/cleanup.rb

```diff
@@ -1,10 +1,11 @@
+require "active_support/core_ext/numeric/time"
```

Comment:
I've been including all of these in `global.rb` to avoid confusion with methods existing depending on the files that have been included already.